### PR TITLE
DTSPO-14613: Pipeline Tests

### PR DESCRIPTION
**Pull request by DJ-Khaled**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPO-14613

### Change description ###

Changed the pipeline tests = true to false in relation to apim.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: azure-pipelines.yaml [hunk 1]
```diff
@@ -91,7 +91,7 @@ parameters:
       - deployment: "apim_appgw"
         environment: "ithc"
         dependsOn: "sbox_apim_appgw"
-        pipeline_tests: true
+        pipeline_tests: flase
```
Comment (endakelly) [inline azure-pipelines.yaml:94]: typo

File: azure-pipelines.yaml [hunk 2]
```diff
@@ -132,30 +132,30 @@ parameters:
...
       - deployment: "apim"
         environment: "prod"
         dependsOn: "sbox_apim"
-        pipeline_tests: true
+        pipeline_tests: flase
       - deployment: "apim"
         environment: "test"
         dependsOn: "sbox_apim"
-        pipeline_tests: true
+        pipeline_tests: flase
```
Comment (endakelly) [inline azure-pipelines.yaml:150]: typo

File: azure-pipelines.yaml [hunk 3]
```diff
@@ -132,30 +132,30 @@ parameters:
...
       - deployment: "apim"
         environment: "stg"
         dependsOn: "sbox_apim"
-        pipeline_tests: true
+        pipeline_tests: false
       - deployment: "apim"
         environment: "prod"
         dependsOn: "sbox_apim"
-        pipeline_tests: true
+        pipeline_tests: flase
```
Comment (endakelly) [inline azure-pipelines.yaml:146]: typo

File: azure-pipelines.yaml (hunk 1 above)
Comment (DJ-Khaled) [reply azure-pipelines.yaml:94]: done

File: azure-pipelines.yaml (hunk 2 above)
Comment (DJ-Khaled) [reply azure-pipelines.yaml:150]: done

File: azure-pipelines.yaml (hunk 3 above)
Comment (DJ-Khaled) [reply azure-pipelines.yaml:146]: done

**Review by endakelly — APPROVED** [review]

**Pull request merged**